sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detectors/apriltag_grid_detector.py

```diff
@@ -66,6 +66,7 @@ class ApriltagGridDetector(BoardDetector):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
+        # cSpell:ignore nthreads
         self.nthreads = Parameter(int, value=8, min_value=1, max_value=16)
```

Comment:
I think `num_threads` is more popular.

Reply:
The apriltag library uses nthreads internally, so I would like to keep it this time

Reply:
In that case, I think it's acceptable.